client/src/components/PictureModal/PictureModalStyles.tsx

```diff
@@ -23,6 +23,7 @@ const useStyles = makeStyles((theme) => ({
     padding: 10,
     '&:hover': {
       background: '#CECECE',
+      cursor: 'pointer',
```

Comment:
As mentioned above, if the element is always clickable no need to set this on hover, it can just be `cursor: 'pointer'`

Reply:
Thanks